### ISSUE TITLE
Unable to click O button, transaction fails

### ISSUE THREAD

**Issue by dhowe** [hicetnunc2000/hicetnunc#560]
Unable to click O button, transaction first shows "likely to fail", then actually fails

- OS: OSX 10.15.7 (19H524)
- Browser: Firefox 88.0

![image](https://user-images.githubusercontent.com/737638/115707311-32aa0f00-a3a1-11eb-8677-d49a9d86b1f6.png)

![Screenshot 2021-04-22 at 7 32 47 PM](https://user-images.githubusercontent.com/737638/115707593-92a0b580-a3a1-11eb-87ce-94295761dd00.png)

**Comment by skenaja** [comment]
Hi @dhowe - that action requires hDAO to upvote (currently 0.0001), and the account's hDAO balance is 0.

![image](https://user-images.githubusercontent.com/510285/115712519-c7464a00-a36c-11eb-9788-0757ec43d2b7.png)

It's probably worth adding validation to this action to check the current balance before allowing upvote.

**Comment by dhowe** [comment]
Yes, I realized this after opening and should have closed the ticket (apologies). But now that's it open, it is a slightly confusing... Validation, as you suggest, would be useful, but perhaps even simpler would be something like mouse-over text that told you what the button does and requires (after reading that it was an 'upvote' my thought was that it worked like 'likes' on other platforms).

Relatedly, I tried to swap some tez for hDAO on a 3rd party platform once I realized it was needed, but wasn't able to make it work. What is the usual way that people on H=N get hDAO to use (and, also, I assume there is a reason that users can't just use tez instead)?
